packages/graphql/src/translate/queryAST/ast/operations/ReadOperation.ts

```diff
@@ -132,8 +134,8 @@ export class ReadOperation extends Operation {
 
         const clause = Cypher.concat(
             ...preSelection,
-            matchClause,
             ...authFilterSubqueries,
+            matchClause,
```

Comment:
Might the `authFilterSubqueries` not require a variable from the `matchClause` out of interest?

Reply:
In theory not, as these should be in the preSelection